### PR TITLE
[FIX] mail: do not show "load more" on temporary thread

### PR DESCRIPTION
**Pull request by phenix-factory**
task-2341690

### CODE REVIEW

File: addons/mail/static/src/components/message_list/message_list.xml [hunk 1]
```diff
@@ -97,9 +97,11 @@
             </div>
         </t>
         <t t-elif="!threadView.threadCache.isAllHistoryLoaded">
-            <a class="o_MessageList_item o_MessageList_loadMore" href="#" t-on-click="_onClickLoadMore" t-ref="loadMore">
-                Load more
-            </a>
+            <t t-if="!threadView.thread.isTemporary">
```
Comment (seb-odoo) [inline addons/mail/static/src/components/message_list/message_list.xml:100]: add the condition on the `elif` just above, no need for a separate `t-if` if there are no further branching

Comment (seb-odoo) [reply addons/mail/static/src/components/message_list/message_list.xml:100]: Also need to update test "base rendering when chatter has no record" to include the check for the absence of this button.

**Review by seb-odoo — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/components/message_list/message_list.xml (hunk 1 above)
Comment (phenix-factory) [reply addons/mail/static/src/components/message_list/message_list.xml]: > add the condition on the elif just above, no need for a separate t-if if there are no further branching

That's what I did in the first place. But it wasn't working. I forgot Owl use `and` for `&&`.

Comment (phenix-factory) [reply addons/mail/static/src/components/message_list/message_list.xml]: done !

File: addons/mail/static/src/components/chatter/chatter_tests.js [hunk 2]
```diff
@@ -134,6 +134,11 @@ QUnit.test('base rendering when chatter has no record', async function (assert)
         "Creating a new record...",
         "should have the 'Creating a new record ...' message"
     );
+    assert.strictEqual(
+        document.querySelectorAll('.o_MessageList_loadMore').length,
+        0,
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chatter/chatter_tests.js:139]: containsNone

Comment (seb-odoo) [reply addons/mail/static/src/components/chatter/chatter_tests.js:139]: `strictEqual` is the old way of doing it that has to not be copied over new assertions

Comment (phenix-factory) [reply addons/mail/static/src/components/chatter/chatter_tests.js]: Done !

File: addons/mail/static/src/components/chatter/chatter_tests.js [hunk 3]
```diff
@@ -134,6 +134,11 @@ QUnit.test('base rendering when chatter has no record', async function (assert)
         "Creating a new record...",
         "should have the 'Creating a new record ...' message"
     );
+    assert.containsNone(
+        document.body,
+        '.o_MessageList_loadMore',
+        "should have the 'load more' button"
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chatter/chatter_tests.js:140]: should NOT have

Comment (phenix-factory) [reply addons/mail/static/src/components/chatter/chatter_tests.js]: Of course, how did I miss that ?!

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**